pkg/providers/cloudstack/cloudstack.go

```diff
@@ -770,7 +773,12 @@ func buildTemplateMapCP(clusterSpec *cluster.Spec, controlPlaneMachineSpec, etcd
 		"externalEtcdVersion":                        bundle.KubeDistro.EtcdVersion,
 		"etcdImage":                                  bundle.KubeDistro.EtcdImage.VersionedImage(),
 		"eksaSystemNamespace":                        constants.EksaSystemNamespace,
-		"auditPolicy":                                common.GetAuditPolicy(),
+	}
+
+	auditPolicy, err := common.GetAuditPolicy(clusterSpec.Cluster.Spec.KubernetesVersion)
+	values["auditPolicy"] = auditPolicy
```

Comment:
can we move this to after the error check? values are not guaranteed to be valid if there is an error